src/lib/review/internalGraphql.ts

```diff
@@ -68,50 +63,6 @@ export const InternalReviewType = new GraphQLObjectType<ReviewFields, Context>({
 });
 
 export const reviewMutations: GraphQLFieldConfigMap<null, Context> = {
-  createReview: {
```

Comment:
Moví esto a submission para solucionar el tema del estado en Relay. La realidad es que se tendría que llamar `addReviewToSubmission` o algo así pero es muy poco usado.

Reply:
https://medium.com/entria/wrangling-the-client-store-with-the-relay-modern-updater-function-5c32149a71ac